mis/data/graphs.py

```diff
@@ -538,3 +540,39 @@ def ingest(self, graph: NXWithPos, device: pl.devices.Device, id: int) -> BaseGr
         pyg.pos = torch.tensor(positions, dtype=torch.float)
 
         return BaseGraph(id=id, device=device, data=pyg)
+
```

Comment:
Note that file `graphs.py` was removed by another PR, but you can recreate it.

Reply:
Ok done. I only added the necessary classes.